src/conn/conn_cache.c

```diff
@@ -155,6 +155,10 @@ __cache_config_local(WT_SESSION_IMPL *session, bool shared, const char *cfg[])
     WT_RET(__wt_config_gets(session, cfg, "cache_max_wait_ms", &cval));
     cache->cache_max_wait_us = (uint64_t)(cval.val * WT_THOUSAND);
 
+    /* Retrieve the timeout value and convert from seconds */
+    WT_RET(__wt_config_gets(session, cfg, "cache_stuck_timeout_s", &cval));
+    cache->cache_stuck_timeout_ms = (uint64_t)(cval.val * WT_THOUSAND);
```

Comment:
Is there a reason you chose different units for the external and internal timeout timers? I'd be inclined to go with `ms` everywhere.

Reply:
We thought from a user perspective it would be more natural to think in seconds. Furthermore, being able to limit the cache from being stuck for a few milliseconds did not seem useful.
I am happy to change to `ms` for consistency though and avoid the conversion.

Reply:
I do prefer milliseconds - for example, it might allow some of our Python tests to be more efficient by dialling down the stuck threshold.

Reply:
I get it, updated to `ms` in this commit: 4b2875d845efe1d7a138629e5c41ea3b35e76c7a